pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>commons-io</artifactId>
             <version>2.4</version>
         </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>1.1.1</version>
```

Comment:
you can use `com.thoughtworks.go.plugin.api.logging.Logger` & remove this dependency.